app/assets/javascripts/admin/admin_ofn.js.coffee

```diff
@@ -8,6 +8,7 @@ angular.module("ofn.admin", [
   "admin.dropdown",
   "admin.products",
   "admin.taxons",
-  "infinite-scroll"
+  "infinite-scroll",
+  "OFNShared"
```

Comment:
I'm not sure if this is the right place for the directive, but this is much more your area of expertise than mine :) It just seems like there's plenty of other cases where we need to share between the two areas, and we haven't had to create this namespace yet. 

(On the other hand, maybe it's not worth the time to fuss over the structure of our Angular code since it's hopefully going away soon anyway...)

Reply:
As far as I understood (not really my area of expertise ...), this is not a directive but a module, which is basically a 
> a container for the different parts of your app – controllers, services, filters, directives, etc.

The aim of this module is precisly to share content (such as directive in our case) between backoffice and frontshop. So I think (but again, maybe I'm wrong) it's the good place to put thinks such like this one.

I fully agree with your comment about Angularjs, but if I've made a big mistake, I'd really happy to learn and change it!